snap/snapcraft.yaml

```diff
@@ -75,6 +80,7 @@ parts:
     plugin: nil
     stage-packages:
       - mysql-server-8.0
+      - util-linux
```

Comment:
what utility in `util-linux` does the mysql-server-deb part need to run?

Reply:
This package contains `setpriv` which is needed for dropping privileges to the `snap_daemon` user